documentation/SwaggerValidationTools.md

```diff
@@ -2,13 +2,6 @@ There are some tools that can help you make sure your specifications conform to
 
 ## Validation tools available online (for MS FTEs)
```

Comment:
Looks like this header should be removed now as well.

Reply:
oops. thank you!
I also went ahead and removed the intro to keep this file only for validation tools